source/particle/interpolator/bilinear_least_squares.cc

```diff
@@ -243,29 +319,26 @@ namespace aspect
+                const unsigned int n_property_components = particle_property_information.n_components();
+                const unsigned int n_internal_components = particle_property_information.get_components_by_field_name("internal: integrator properties");
+                std::vector<std::string> split = Utilities::split_string_list(prm.get("Use linear least squares limiter"));
+                if (split.size() == 1)
+                  {
+                    use_linear_least_squares_limiter = ComponentMask(n_property_components, string_to_bool(split[0]));
+                  }
+                else if (split.size() == n_property_components - n_internal_components)
                   {
-                    global_maximum_particle_properties = Utilities::string_to_double(Utilities::split_string_list(prm.get("Global particle property maximum")));
-                    global_minimum_particle_properties = Utilities::string_to_double(Utilities::split_string_list(prm.get("Global particle property minimum")));
-
-                    const Postprocess::Particles<dim> &particle_postprocessor =
-                      this->get_postprocess_manager().template get_matching_postprocessor<const Postprocess::Particles<dim>>();
-                    const auto &particle_property_information = particle_postprocessor.get_particle_world().get_property_manager().get_data_info();
-                    const unsigned int n_property_components = particle_property_information.n_components();
-                    const unsigned int n_internal_components = particle_property_information.get_components_by_field_name("internal: integrator properties");
-
-                    // Check that if a global limiter is used, we were given the minimum and maximum value for each
-                    // particle property that is not an internal property.
-                    AssertThrow(global_minimum_particle_properties.size() == n_property_components - n_internal_components,
-                                ExcMessage("Make sure that the size of list 'Global minimum particle property' "
-                                           "is equivalent to the number of particle properties."));
-
-                    // Check that if a global limiter is used, we were given the minimum and maximum value for each
-                    // particle property that is not an internal property.
-                    AssertThrow(global_maximum_particle_properties.size() == n_property_components - n_internal_components,
-                                ExcMessage("Make sure that the size of list 'Global maximum particle property' "
-                                           "is equivalent to the number of particle properties."));
+                    std::vector<bool> parsed(n_property_components, false);
+                    for (unsigned int i = 0; i < split.size(); i++)
+                      parsed[i] = string_to_bool(split[i]);
```

Comment:
You can use modern c++ feature to make this a bit easier:
```suggestion
                    std::vector<bool> parsed;
                    for (const auto &component: split)
                      parsed.push_back(string_to_bool(component));
```